test/extensions/filters/listener/http_inspector/http_inspector_fuzz_test.cc

```diff
@@ -0,0 +1,32 @@
+#include "extensions/filters/listener/http_inspector/http_inspector.h"
+
+#include "test/extensions/filters/listener/common/fuzz/listener_filter_fuzzer.h"
+#include "test/extensions/filters/listener/common/fuzz/listener_filter_fuzzer.pb.validate.h"
+#include "test/fuzz/fuzz_runner.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace ListenerFilters {
+namespace HttpInspector {
+
+DEFINE_PROTO_FUZZER(const test::extensions::filters::listener::FilterFuzzTestCase& input) {
+
+  try {
+    TestUtility::validate(input);
+  } catch (const ProtoValidationException& e) {
+    ENVOY_LOG_MISC(debug, "ProtoValidationException: {}", e.what());
+    return;
+  }
+
+  Stats::IsolatedStoreImpl store;
+  ConfigSharedPtr cfg = std::make_shared<Config>(store);
+  auto filter = std::make_unique<Filter>(cfg);
+
+  ListenerFilterFuzzer fuzzer;
```

Comment:
This is the place I mentioned: would it possible to make this fuzzer static?  Maybe we could instantiate some member objects only once, and set up some mock ON_CALL only once. In this way, we only need to set up some behaviors multiple times when this kind of behavior must be changed during each execution.

Reply:
Yup, that was the original intent - instantiate the fuzzer once, and then be able to call fuzz() repeatedly for different inputs and filters. Changed the fuzzer to be static in the latest commit.